org.eclipse.xtext/src/org/eclipse/xtext/generator/trace/node/TracedAccessors.xtend

```diff
@@ -89,6 +89,8 @@ class TracedAccessorsProcessor extends AbstractClassProcessor {
 		if (declaration.static)
 			return false
 		val n = it.declaration.simpleName
+		if (n == 'getClass')
+			return false
```

Comment:
I think it would be better to check whether the declaring type of the method is `Object`.